### DISCUSSION TITLE
TF 2.3 meeting 19th Jan 2023

### DISCUSSION THREAD

**Discussion by LucyKershaw** [OSIPI/DCE-DSC-MRI_CodeCollection#121] [General]
Minutes from the meeting attached

Lucy

[Minutes_Jan23.pdf](https://github.com/OSIPI/DCE-DSC-MRI_CodeCollection/files/10499044/Minutes_Jan23.pdf)

**Comment by stadmill** [comment]
Thanks Lucy! Everyone, please check your emails for the google docs link to the roadmap - to be completed by 31 Jan!!!
